canonical-web-and-design/jaas-dashboard
# Update IP address on register controllers

Fixes #697

## Done

Update IP address on register controllers

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check
